api/gen_server.go

```diff
@@ -1,6 +1,6 @@
 // Package api provides primitives to interact with the openapi HTTP API.
 //
-// Code generated by github.com/deepmap/oapi-codegen version v1.12.4 DO NOT EDIT.
+// Code generated by github.com/deepmap/oapi-codegen version v1.10.0 DO NOT EDIT.
```

Comment:
@clintonium-119 can you regenerate the spec with oapi-codegen version v1.12.4. You're downgrading to v.1.10.X in this PR.

I think you need to install go1.18 locally and pull the latest oapi codegen.

Reply:
Ah.  Didn't notice this.  I went with v1.10.0 since that's what was in the go.mod  (I had been at 1.9.0)
Will update this shortly.